toolkit/scripts/toolchain/container/toolchain-sha256sums

```diff
@@ -26,7 +26,8 @@ f9b55476def44fc7c459b2537d17dbc731e36ed5d416af7ca0b1e2e676f8aa04  dri2proto-2.8.
 01be49d70200518b9a6b297131f6cc71f4ea2de17436896af153226a774fc074  dri3proto-1.0.tar.bz2
 9bf7200d2737ed13f50a080af285c11529f91b088d84ecb71aae9fac58a8fbee  e2fsprogs-1.44.6.tar.gz
 dc8d3e74ab209465e7f568e1b3bb9a5a142f8656e2b57d10049a73da2ae6b5a6  elfutils-0.185.tar.bz2
-17b43c2716d521369f82fc2dc70f359860e90fa440bea65b3b85f0b246ea81f2  expat-2.2.6.tar.bz2
+2f9b6a580b94577b150a7d5617ad4643a4301a6616ff459307df3e225bcfbf40  expat-2.4.1.tar.bz2
+49a7da83b0bdd9f46d04a04deec19c7767bb9a323e40c4781f89caf760b92c34  expect5.45.4.tar.gz
```

Comment:
This line seems irrelevant?

Reply:
You're correct, not sure where that came from when merging dev in. Removing, thanks for catching that.